pkg/bpftrace/build-release.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
can you use /bin/sh - this does not use any bash features...

Reply:
Thanks. Fixed in the next commit.